### PR TITLE
Fixes chameleon mutation

### PR DESCRIPTION
**Pull request by Razharas**
Fixes chameleon mutation
Fix https://github.com/tgstation/-tg-station/issues/8404

### CODE REVIEW

**Comment by Kromgar** [comment]
Wow it only took like 3 months

File: code/datums/mutations.dm [hunk 1]
```diff
@@ -375,16 +378,17 @@
 	lowest_value = 256 * 14
 	text_gain_indication = "<span class='notice'>You feel one with your surroundings.</span>"
 	text_lose_indication = "<span class='notice'>You feel oddly exposed.</span>"
-	var/last_location
+
+/datum/mutation/human/chameleon/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	owner.alpha = 204  //famous 255 * 0.8, yes its magic number, no i wont make it a define
```
Comment (RemieRichards) [inline code/datums/mutations.dm]: make it a define.

**Comment by Supermichael777** [comment]
does this mean mutant race is finally untucked enough to jam ind dismemberment? also when

**Comment by Razharas** [comment]
If anything this will conflict with ricotezs dismemberment a bit

**Comment by Ricotez** [comment]
not really, I also modify human_movement but in another place so even if it causes conflicts they're easy to resolve

**Pull request merged**